DeskKitSwiftExample/DeskKitSwiftExample/SplitViewController.swift

```diff
@@ -0,0 +1,263 @@
+            }) { (emailAction) in
+                self.alertControllerDidTapEmailUs()
+        }
+        
+        let contactUsButton = toolbarItems[contactUsButtonIndex]
+        contactUsSheet.popoverPresentationController?.barButtonItem = contactUsButton
+        present(contactUsSheet, animated: true, completion: nil)
+    }
+    
+    private func alertControllerDidTapEmailUs() {
+        let contactUsVC = DKSession.sharedInstance().newContactUsViewController()
+        contactUsVC.delegate = self
+        
+        /*
+        // Example of adding custom fields
+        // 1. Grab initial custom fields populated from DeskKitSettings.plist
+        let customFields = NSMutableDictionary(dictionary:contactUsVC.customFields)
+        // 2. Add your own dynamic custom fields.
+        customFields.addEntries(from: dynamicCustomFields)
+        // 3. Assign back to property.
+        contactUsVC.customFields = customFields as [NSObject : AnyObject];
+        */
+        
+        // Configure additional properties of DKContactUsViewController here
+        
+        
+        let navVC = UINavigationController(rootViewController: contactUsVC)
+        navVC.modalPresentationStyle = .pageSheet
+        present(navVC, animated: true, completion: nil)
+    }
+    
+    // MARK: Custom Field Example
+    
+    var dynamicCustomFields: [NSObject: AnyObject] {
+        return [
+            "my_case_boolean_custom_field" : true,
+            "my_case_date_custom_field" : NSDate().stringWithISO8601Format(),
+            "my_case_list_custom_field" : "C",
+            "my_case_number_custom_field" : 45, // Integer
+            "my_case_text_custom_field" : "value1"
+        ]
+    }
+    
+    // MARK: DKContactUsViewControllerDelegate
+    
+    func contactUsViewControllerDidSendMessage(_ viewController: DKContactUsViewController!) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func contactUsViewControllerDidCancel(_ viewController: DKContactUsViewController!) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: DKTopicsViewControllerDelegate
+    
+    func topicsViewController(_ topicsViewController: DKTopicsViewController!, didSelect topic: DSAPITopic!, articlesTopicViewModel: DKArticlesTopicViewModel!) {
+        let controller = newArticlesViewController()
+        
+        controller.delegate = self
+        controller.setViewModel(articlesTopicViewModel, topic: topic)
+        
+        masterNavigationController?.pushViewController(controller, animated: true)
+    }
+    
+    func topicsViewController(_ topicsViewController: DKTopicsViewController!, didSelectSearchedArticle article: DSAPIArticle!) {
+        showArticle(article: article)
+    }
+    
+    // MARK: DKArticlesViewControllerDelegate
+    
+    func articlesViewController(_ articlesViewController: DKArticlesViewController!, didSelectSearchedArticle article: DSAPIArticle!) {
+        showArticle(article: article)
```

Comment:
This might cause crash when articles is nil. I think we should add guard statement here.